samples/nrf9160/lwm2m_carrier/src/main.c

```diff
@@ -57,9 +67,13 @@ int lwm2m_carrier_event_handler(const lwm2m_carrier_event_t *event)
 	case LWM2M_CARRIER_EVENT_READY:
 		printk("LWM2M_CARRIER_EVENT_READY\n");
 		break;
-	case LWM2M_CARRIER_EVENT_DEFERRED:
+	case LWM2M_CARRIER_EVENT_DEFERRED: {
+		lwm2m_carrier_event_deferred_t *devt = event->data;
 		printk("LWM2M_CARRIER_EVENT_DEFERRED\n");
+		printk("Reason %u, timeout %d seconds\n",
```

Comment:
We print a string representation of the error so we should probably also print a string representation of reason here.